src/tools/compiletest/src/lib.rs

```diff
@@ -538,6 +492,8 @@ pub fn opt_str2(maybestr: Option<String>) -> String {
 
 /// Called by `main` after the config has been parsed.
 pub fn run_tests(config: Arc<Config>) {
+    debug!(?config, "run_tests");
```

Comment:
Discussion: hm, I _think_ it was originally intended so the config gets logged when run in CI?

Reply:
If we suspect it's still useful, I can add in a `println!("{config:#?}")` to just dump the whole config.

(It's *mostly* redundant with the compiletest command-line arguments that `./x --verbose` will print out anyway, but I suppose that seeing the post-parse config could have some value.)

Reply:
Hm right. I think we might want to dump the post-parse config under CI env, but not necessary for this PR.